src/layout/UserMenu.vue

```diff
@@ -12,122 +12,24 @@
-                Sign up for NimbusImage!
-              </h2>
-              <p class="text-subtitle-1">
-                Create a new account to get started.
-              </p>
-            </div>
-            <v-form @submit.prevent="signUp" class="my-8">
-              <v-text-field
-                v-if="!isDomainLocked"
-                v-model="domain"
-                name="domain"
-                label="Girder Domain"
-                required
-                prepend-icon="mdi-domain"
-              />
-              <v-text-field
-                v-model="signupUsername"
-                name="username"
-                label="Username"
-                required
-                prepend-icon="mdi-account"
-                autocomplete="username"
-              />
-              <v-text-field
-                v-model="signupEmail"
-                name="email"
-                label="Email"
-                required
-                prepend-icon="mdi-email"
-                autocomplete="email"
-              />
-              <v-text-field
-                v-model="signupFirstName"
-                name="firstName"
-                label="First Name"
-                required
-                prepend-icon="mdi-account"
-                autocomplete="given-name"
-              />
-              <v-text-field
-                v-model="signupLastName"
-                name="lastName"
-                label="Last Name"
-                required
-                prepend-icon="mdi-account"
-                autocomplete="family-name"
-              />
-              <v-text-field
-                v-model="signupPassword"
-                name="password"
-                type="password"
-                label="Password"
-                required
-                prepend-icon="mdi-lock"
-                autocomplete="new-password"
-              />
-              <div class="d-flex flex-column">
-                <v-btn type="submit" color="primary"> Sign up </v-btn>
-                <v-btn text class="align-self-end my-2" @click="switchToLogin">
-                  Back to login
-                </v-btn>
-              </div>
-            </v-form>
-          </template>
-          <v-alert :value="!!errorMessage" type="error">
-            {{ errorMessage }}
-          </v-alert>
-          <v-alert :value="!!successMessage" type="success">
-            {{ successMessage }}
-          </v-alert>
+          <div class="text-center mb-8">
+            <h2 class="text-h5 font-weight-bold mb-2">
+              Welcome to NimbusImage!
+            </h2>
+            <p class="text-subtitle-1">
+              A cloud-based image analysis platform from the Raj Lab at the
+              University of Pennsylvania and Kitware
+            </p>
+          </div>
+          <v-text-field
+            v-if="!isDomainLocked"
+            v-model="domain"
+            name="domain"
+            label="Girder Domain"
```

Comment:
I have noticed that when you log out, there are two "Girder Domain" fields visible.

<img width="383" alt="image" src="https://github.com/user-attachments/assets/a3ff95de-12d8-4189-b199-df4dd13b2566" />

Reply:
Whoops, my bad, I removed it.
Now it's up to UserMenu.vue to handle that field, which allows me to simply swap the component UserMenuLoginForm.vue without having to care about the domain